cmd/playground/interactive_window.go

```diff
@@ -0,0 +1,213 @@
+package playground
+
+import (
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/charmbracelet/bubbles/textinput"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/odpf/optimus/job"
+)
+
+// model contains a cursor to keep track of the pointer, two input fields for taking Size and Offset as input
+// model also contains a state which maintains both versions of windows and the sechdule time
+type model struct {
+	cursor      string
```

Comment:
using `string` for `cursor` field is a bit confusing. we have to guess what is inside this `cursor`. this is also related when we use it under `genarateFields`. we don't know what is this `position` gonna be? you can create a different type for this, like `cursorType` or something.